src/platforms/stm32/src/lib/sys.c

```diff
@@ -119,7 +119,7 @@ void platform_defaultatoms_init(GlobalContext *glb)
 void sys_enable_core_periph_clocks()
 {
     uint32_t list[] = GPIO_CLOCK_LIST;
-    for (int i = 0; i < sizeof(list) / sizeof(list[0]); i++) {
+    for (uint i = 0; i < sizeof(list) / sizeof(list[0]); i++) {
```

Comment:
I think size_t is more natural here, rather than uint.
Hence `uint i` -> `size_t i`.

Reply:
fixed.